MODULES/EDGE.py

```diff
@@ -2887,9 +2905,16 @@ def dered(self, Av, law, datapath, Av_unc = 0, flux=1, lpath=commonpath, rv = No
         """
 
         # Read in the dereddening laws pickle. The default is whereever you keep EDGE.py, but you can move it.
-        extPickle = open(lpath + 'ext_laws.pkl', 'rb')
-        extLaws   = cPickle.load(extPickle, encoding = 'latin1')
-        extPickle.close()
+        original = lpath + 'ext_laws.pkl'
+        destination = "data_unix.pkl"
+        content = ''
+        with open(original, 'rb') as infile:
+            content = infile.read()
+        with open(destination, 'wb') as output:
```

Comment:
Why is this necessary? Did the format of ext_laws.pkl get corrupted somehow? If so, it'd be better to replace the file itself in the repo instead of doing this every time.

Reply:
I remember having this issue when I first started working with EDGE. I think this might be a system-specific issue. On my Windows desktop cPickle does not recognize DOS lines so it has to be converted to Unix somehow. A solution might be wrapping these lines in a try-except clause?  It is possible that a Mac or Linux user might not be able to use the Unix format that I adopted here and have to use the original. So it might be better that the script only converts the pickle file to Unix if an exception occurs.

Reply:
I think Mac and Linux should have no problem with Unix end-of-line characters (in fact, I would have expected an issue with DOS EOL characters instead). Let me test this in my Mac, and if it works (as it should), we can just replace  ext_laws.pkl with Unix EOL characters.

Reply:
I just tested this and it works fine. We can just replace the ext_laws.pkl file with the one with Unix EOL characters. Just in case, we can also keep the original one as ext_laws_DOS.pkl for example. I don't think anyone will need it, but just in case.